source/appendices/e-audit-requirements-and-tests/05-tr-book-tests.rst

```diff
@@ -0,0 +1,145 @@
+.. The COUNTER Code of Practice Release 5 © 2017-2021 by COUNTER
+   is licensed under CC BY-SA 4.0. To view a copy of this license,
+   visit https://creativecommons.org/licenses/by-sa/4.0/
+
+E.5 Title Report Audit Tests for Books and Reference_Works
```

Comment:
The Title Report tests still only cover book and journal content. What about platforms that only have other content (e.g. just Patents, Reports or Standards)? On platforms with a mixture of book/journal and other content the latter is only tested for PR and DR/IR (if provided).